<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskItemWidget</class>
 <widget class="QWidget" name="TaskItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="filenameLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>5</y>
     <width>440</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>filename:size</string>
   </property>
  </widget>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="elapsedTimelabel">
   <property name="geometry">
    <rect>
     <x>634</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用时:0:0:0</string>
   </property>
  </widget>
  <widget class="QLabel" name="remainTimeLabel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>剩余时间:-:-:-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseBtn">
   <property name="geometry">
    <rect>
     <x>907</x>
     <y>20</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtn">
   <property name="geometry">
    <rect>
     <x>967</x>
     <y>20</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="speedLabel">
   <property name="geometry">
    <rect>
     <x>494</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>速度:0.00MB/s</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>1030</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pauseBtn</sender>
   <signal>clicked()</signal>
   <receiver>TaskItemWidget</receiver>
   <slot>toggleSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopBtn</sender>
   <signal>clicked()</signal>
   <receiver>TaskItemWidget</receiver>
   <slot>stopSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleSlot()</slot>
  <slot>stopSlot()</slot>
 </slots>
</ui>
